frontend/src/v5/ui/v4Adapter/overrides/cards/tickets.overrides.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { css } from 'styled-components';
+
+export default css`
+	#tickets {
```

Comment:
I am not sure this code should be here. The `*.overrides.ts` files are to override v4 styling, but in this case there is nothing to override as we do not support custom tickets in v4